packages/libs/lambda-at-edge/tests/default-handler/default-handler-with-basepath.test.ts

```diff
@@ -109,8 +109,6 @@ describe("Lambda@Edge", () => {
       it.each`
         path                                                        | expectedPage
         ${"/basepath"}                                              | ${"/index.html"}
-        ${"/basepath/"}                                             | ${"/index.html"}
```

Comment:
Trailing slash should be redirected to non-trailing slash instead of going to index.html page